docs/content/user-guide/settings/publishable-api-keys.mdx

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 4
+---
+
+import UiIcon from '@site/src/components/UiIcon';
+
+# Manage Publishable API Keys
+
+In this document, you’ll learn how to manage publishable API keys in Medusa Admin.
+
+## Overview
+
+Publishable API keys are used for development purposes. They are used to define the resources that an API key is associated with and can access.
```

Comment:
Actually that wasn't what I meant here. As this is a user guide, it's aimed towards non-technical users, so my point was it would be used by developers. However, I can see where your comment comes from and how it can cause confusion, so I'll fix it!